### ISSUE TITLE
Check on performance with JAWS

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#118]
There has been work on description in this sim, but it has been put on hold. RIAW with description is in some final stages of RC testing. In the testing, it was determined that performance with JAWS was poor and caused alerts to not be provided after moving a slider. See https://github.com/phetsims/resistance-in-a-wire/issues/165.

The optimization there was to use CanvasNodes for the dots, which improved performance with JAWS a bit.  We should see if something similar is needed in this sim. I think the order of priority should be

1) Verify that https://github.com/phetsims/resistance-in-a-wire/issues/165 is still an issue in this sim.
2) If it is, see if features in utteranceQueue like delayTime help with this issue.
3) See if we can re-structure alerts to use aria-valuetext instead of aria-live.
4) Implement the dots with CanvasNode like in RIAW

From a development perspective, both 2 and 3 would be less time intensive than 4 by a long shot. But 3 might have impacts on the user experience(?).

**Comment by jessegreenberg** [comment]
Because of performance issues with JAWS in IE11, we have switched to JAWS + Chrome. Hopefully, this optimization is no longer needed because of this change. Assigning to myself to investigate.

**Comment by jessegreenberg** [comment]
The performance issues in https://github.com/phetsims/resistance-in-a-wire/issues/165 very apparent when testing on my Windows 10 laptop. If they are no longer present on that machine I think this can be closed.

**Comment by jessegreenberg** [comment]
Checked in #123, performance with JAWS in Chrome is very nice. This issue can be closed.
